shelley/chain-and-ledger/executable-spec/delegation.cabal

```diff
@@ -99,6 +99,12 @@ test-suite delegation-test
       -Wincomplete-record-updates
       -Wincomplete-uni-patterns
       -Wredundant-constraints
+      -- We set a bound here so that we're alerted of potential space
+      -- leaks in our generators (or test) code.
+      --
+      -- The 4 megabytes stack bound and 50 megabytes heap bound were
+      -- determined ad-hoc.
```

Comment:
isn't that 150MiB ?

Reply:
I didn't change the comment when in `cs-ledger.cabal` :man_facepalming: Thanks for catching that :)

Reply:
It's worse that I copied the comment without catching this. ￼ Thank you!